autovalet_system/src/autovalet_system.py

```diff
@@ -133,22 +144,29 @@ def init_detector(self, colorInfo_topic, laneCloud_topic, egoLine_topic):
                           egoLine_topic,
                           hlsBounds,
                           lineParams,
+                          self.sim,
                           debug=False)
 
         self.ld_init = True
 
         return LD
 
+    # def published(self):
+
+
     # callback for image messages. this is what keeps the system moving forward, as processState gets
     # called everytime we get a new img
     def registered_image_callback(self, color_msg, depth_msg):
+        self.frame_count += 1
+
         # cvBridge image
         self.color_img = self.bridge.imgmsg_to_cv2(color_msg, desired_encoding="bgr8")
         self.depth_img = self.bridge.imgmsg_to_cv2(depth_msg, desired_encoding="passthrough")
 
         self.depth_frame_id = depth_msg.header.frame_id
-
         # if we're not in the PARK/FINSIH state AND the lane detector has been successfully initialized, detect the lane and publish
```

Comment:
put this comment right before if statement (a few lines down)

Reply:
yes sir